### PR TITLE
Fix `page-title` of science blog

### PR DESCRIPTION
**Pull request by Ein-Tim**
This commit fixes the `page-title` of the English science blog.

### CODE REVIEW

File: src/pages/en/science/index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 lang_de: false
-page-title: Open-Source Project Corona-Warn-App - Blog
+page-title: Open-Source Project Corona-Warn-App - Science Blog
 page-description: Blog for the Corona-Warn-App. Here you get current information about the development of the Corona-Warn-App and all important updates.
```
Comment (MikeMcC399) [inline src/pages/en/science/index.html:4]: The page-description is also incorrect. Will you change this as well?

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Please review also `page-description:`.

File: src/pages/en/science/index.html (hunk 1 above)
Comment (Ein-Tim) [reply src/pages/en/science/index.html:4]: I‘ll change this as soon as I have the time. Thanks for pointing that out.

**Comment by Ein-Tim** [comment]
@MikeMcC399 Changed!

**Review by GisoSchroederSAP — APPROVED** [review]
Please have a look one more time to the phrase "und alle wichtigen Update." In my browser it looks like the last 's' for the plural "Updates" is missing. 

Other than that, fine, please go ahead.

**Review by MikeMcC399 — APPROVED** [review]
LGTM!

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 

> Please have a look one more time to the phrase "und alle wichtigen Update." In my browser it looks like the last 's' for the plural "Updates" is missing.

When you review the PR, I recommend you go to "Files changed", then click on the gear wheel and select the "Split" view not the "Unified" view. The lines of source for the web repo are often very long and the split view wraps the lines automatically. The unified view doesn't wrap so you may need to use the scroll bar to review all characters.

If you view the changes in this PR and do not scroll then indeed the word "Update" is missing the "s" until you scroll right. As @Ein-Tim said though, this text is in the portion being removed (marked with a "-" at the beginning of the line and coloured red.

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim  @MikeMcC399 Thanks!

**Pull request merged**